tests/integration/targets/vm_git_issues/tasks/issue_15_tiering_priority.yml

```diff
@@ -0,0 +1,155 @@
+# ----------------------------------Cleanup------------------------------------------------------------------------
+- name: Delete XLAB-vm
+  scale_computing.hypercore.vm: &delete-XLAB-vm
+    vm_name: "{{ item }}"
+    state: absent
+  loop:
+    - XLAB-test-tiering-priority
+    - XLAB-test-tiering-priority-running
+
+# ----------------------------------Setup-----------------------------------------------------------------------------
+- name: Create a source VM
+  scale_computing.hypercore.api:
+    action: post
+    endpoint: /rest/v1/VirDomain
+    data:
+      dom:
+        name: XLAB-test-tiering-priority
+        tags: Xlab,integration
+        mem: 512100100
+        numVCPU: 2
+        blockDevs:
+          - type: VIRTIO_DISK
+            capacity: 8100100100
+            name: jc1-disk-0
+            slot: 0
+      options:
+        attachGuestToolsISO: false
+  register: vm_created
+- ansible.builtin.assert:
+    that:
+      - vm_created is changed
+      - vm_created is succeeded
+
+- name: Retrieve XLAB-test-tiering-priority
+  scale_computing.hypercore.vm_info:
+    vm_name: XLAB-test-tiering-priority
+  register: source_info
+- ansible.builtin.assert:
+    that:
+      - source_info.records | length == 1
+      - source_info.records.0.power_state == "stopped"
+      - source_info.records.0.disks.0.tiering_priority_factor == 0
```

Comment:
Here I think - maybe tiering_priority_factor should be 4 by default?

Reply:
Yes, I think so, I'm not exactly sure, if a newly created VM has tiering_priority_factor set to 4 as default or 0.

Reply:
This is fixed in test now. Problem is HC3 returs 0 for first 5 secods, after 30 sec or more it does return expected 8